automation/check-patch.e2e.sh

```diff
@@ -18,7 +18,6 @@ main() {
     cd ${TMP_PROJECT_PATH}
 
     echo 'Run golint'
```

Comment:
We should remove this message as well, or run run a different linter, like go vet instead?

Reply:
Let's just run `make format`